### ISSUE TITLE
sitewide, components [COGNITION]: Focus outline SHOULD be consistent throughout

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#7295]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

```diff
! Team affected: VSA Public Websites; Project: Appeals Modernization (AMA) phase 1
```

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

During the VA 508 Office review, on https://va.gov/disability/, the Focus for “Eligibility”, “How to file a claim” is an underline where the rest of the screen the focus is a yellow box. They recommend consistency. Upon investigation, this happens on Windows 10, with IE11 (the VA 508 Office's testing suite).

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As an IE11 keyboard user, I want to navigate the content and consistently identify what is in focus.

## Environment

* Operating System: Windows 10
* Browser: IE11
* Server destination: production

## Steps to Recreate

1. Enter `https://va.gov/disability/` in browser
2. Navigate by keyboard through the focus areas
3. Notice that most focus areas have a yellow outline
4. Verify that the Eligibility and How to file a claim linked headings display an underline, and a faint yellow line on the left side

## WCAG or Vendor Guidance (optional)

* [W3C Understanding WCAG 2.0 - Consistent Identification SC 3.2.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-consistent-functionality.html)

## Screenshots or Trace Logs
If you look closely at this screenshot from Windows 10 + IE11, there's a tiny bit of a yellow line on the left side. I suspect there's something with the offset, padding, margin, or possibly the border-box properties.
![image (1)](https://user-images.githubusercontent.com/57469/77762343-c333e580-700f-11ea-996e-574833281649.png)

**Comment by jenniferlee-dsva** [comment]
completed, all links now have yellow box focus state. closing,
